pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -578,9 +539,15 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         });
         setupDDPConfigurationLookup(cfg.getString(ApplicationConfigConstants.DDP));
 
-        AuthenticationRoute authenticationRoute =
-                new AuthenticationRoute(auth0Util, jwtSecret, cookieSalt, cookieName, userUtil, cfg.getString("portal.environment"));
-        post(UI_ROOT + RoutePath.AUTHENTICATION_REQUEST, authenticationRoute, new JsonTransformer());
+        AuthenticationRoute authenticationRoute = new AuthenticationRoute(auth0Util,
```

Comment:
This object is not being used.

Reply:
Actually, I am not seeing the AuthenticationRoute being used at all. Did we set a route path for it somewhere I am missing?

Reply:
Sorry, My bad! It is back on L563!